sdk/go/common/apitype/history.go

```diff
@@ -84,6 +84,7 @@ const (
 // Should generally mirror backend.UpdateInfo, but we clone it in this package to add
 // flexibility in case there is a breaking change in the backend-type.
 type UpdateInfo struct {
+	UpdateVersion int `json:"updateVersion"`
```

Comment:
I'd suggest we just keep this as version to be consistent with the other places in the code we refer to this value. 

```suggestion
	Version int `json:"version"`
```